plutus-core/untyped-plutus-core/test/Spec.hs

```diff
@@ -10,29 +11,33 @@ import Evaluation.Builtins (test_builtins)
 import Evaluation.Debug (test_debug)
 import Evaluation.FreeVars (test_freevars)
 import Evaluation.Golden (test_golden)
-import Evaluation.Machines
+import Evaluation.Machines (test_budget, test_machines, test_tallying)
 import Evaluation.Regressions (schnorrVerifyRegressions)
 import Flat.Spec (test_flat)
 import Generators (test_parsing)
+import Transform.CaseOfCase.Test (test_caseOfCase)
 import Transform.Simplify (test_simplify)
 
 import Test.Tasty
 
 main :: IO ()
 main = do
   setLocaleEncoding utf8
-  defaultMain $ testGroup "Untyped Plutus Core"
-    [ test_machines
-    , test_builtins
-    , test_budget
-    , test_golden
-    , test_tallying
-    , test_simplify
-    , test_debruijn
-    , test_freevars
-    , test_parsing
-    , test_debug
-    , test_flat
-    , schnorrVerifyRegressions
-    , evalOrder
-    ]
+  defaultMain $
+    testGroup
+      "Untyped Plutus Core"
```

Comment:
I like your layout more for aesthetic reasons, but the one I've committed is done by fourmolu and I just go with it, because I like to not care about formatting for a while and then re-format the whole file and make it uniform afterwards.

Reply:
Fair enough. I'm surprised that fourmolu does that, I thought it didn't add newlines or something.